1.8/administration/access-node/tunnel.md

```diff
@@ -114,16 +114,33 @@ DC/OS Tunnel provides you with full access to the DNS, masters, and agents from
 * [SSH access](/docs/1.8/administration/access-node/sshcluster/) (key authentication only).
 * [The OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html) for VPN functionality.
 
-**Note:** Only Linux and OS X are supported currently.
+**Note:** Only Linux and macOS are supported currently.
+
+## Example Application
+
+All examples will refer to this sample application:
+* Service Name: `myapp`
+* Group: `mygroup`
+* Port: `555`
+ * Port Name: `myport`
+ * Load-balanced
+* Running on overlay network
+
+`myapp` is a web server listening on port `555`. We'll be using `curl` as
+our client application.
 
 ##  SOCKS
 Run the following command from the DC/OS CLI:
```

Comment:
We put the blurb on what's going on up above, this is just how to use it.

Since the example I'm using is a web server I suppose people would expect to see their web page here. I'll make it clearer what to expect up in the "Example Application" description.